docs/credential-plugins.rst

```diff
@@ -379,3 +380,66 @@ Below shows an example of a configured Thycotic Secret Server credential.
 
 .. image:: _static/images/credentials-create-thycotic-server-credential.png
    :alt: Example new Thycotic Secret Server credential lookup dialog
+
+
+.. _ug_credentials_github_app_lookup:
+
+GitHub App Token Lookup
+~~~~~~~~~~~~~~~~~~~~~~~
+
+This plugin allows a GitHub app token to be used as a credential input source to pull secrets from GitHub App. AWX uses existing GitHub auth from organizations' GitHub repos. Refer to `Generating an installation access token for a GitHub App <https://docs.github.com/en/apps/creating-github-apps/authenticating-with-a-github-app/generating-an-installation-access-token-for-a-github-app>`_ for more detail.
+
+1. Create a lookup credential that stores your secrets. See TBD for detail.
+..(link available in separate PR that has this information).
+
+2. When **GitHub App Installation Access Token lookup** is selected for **Credential Type**, provide the following attributes to properly configure your lookup:
+
+- **GitHub App ID** (required): provide the app ID used for communicating with your GitHub App
```

Comment:
GitHub App ID isn't used to communicate with the GitHub App. We *are* a GitHub App for all intents and purposes. And us (this awx plugin aka a GitHub App) talks to API enpoints that GitHub hosts.
```suggestion
- **GitHub App ID** (required): type in the App ID provided by your instance of GitHub, this is what we will authenticate as
```